packages/node_modules/@ciscospark/widget-recents/src/components/NoSpaces.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import styles from './NoSpaces.css';
+
+const propTypes = {
+  title: PropTypes.string.isRequired,
+  emptyMessage: PropTypes.string.isRequired
+};
+
+const defaultProps = {
+
+};
+
+function NoSpaces(props) {
+  const {
+    title,
+    emptyMessage
+  } = props;
+
+  return (
+    <div className={classNames('ciscospark-recents-widget', styles.noSpacesWrapper)}>
```

Comment:
`ciscospark-recents-widget` is the class name of the main recents widget. I would expect it to be `ciscospark-recents-widget-no-spaces` perhaps.

Reply:
yeah, that's what i had at first i think i was just testing why the recents tests weren't working, will put it back